### DISCUSSION TITLE
How can we check particular SoC is supported by Zephyr?

### DISCUSSION THREAD

**Discussion by KZONS** [zephyrproject-rtos/zephyr#69371] [Q&A] [answered]
How can we check particular SoC is supported by Zephyr?
Although "Supported Boards" are defined in Zephyr official page, "Supported SoCs" are not defined.

According to "Supported Boards", some of SoC can be supported because boards apply them.
But about SoCs that are not applied by particular boards, we cannot judge if the SoCs are applied or unapplied.

ex. NXP MIMXRT1171
According to "Supported Boards", NXP MIMXRT1170-EVKB applies "MIMXRT1176DVMAA MCU"
Zephyr OS mentions NXP MIMXRT1171 in "Kconfig.soc", "MIMXRT1171.h" and some other files.
But Zephyr official page doesn't mention NXP MIMXRT1171.
How can we judge NXP MIMXRT1171 is supported by Zephyr?

**Comment by yashi** [comment, marked as answer]
I think it largely depends on your definition of _supported_.

> But about SoCs that are not applied by particular boards, we cannot judge if the SoCs are applied or unapplied.

We can't _support_ a SoC without a board.  Thus, we list "supported boards" instead of "supported SoC".

I don't even know the difference between 1171 and 1170.  But just looking at the datasheets, it seems MIMXRT1171DVMAA is a chip lacks a few features of MIMXRT1176DVMAA.

**Reply by KZONS** [reply]
Thanks for reply.
I understood that a SoC can't be supported without a board.
And SoC support files of MIMXRT1171DVMAA can be made based on the files of MIMXRT1176DVMAA  reducing features which MIMXRT1171DVMAA doesn't apply. That's because MIMXRT1176DVMAA is upward compatible with MIMXRT1171DVMAA and MIMXRT1176DVMAA is supported by Zephyr.
Is my understanding correct?

**Reply by yashi** [reply]
That's what I understand.

**Reply by KZONS** [reply]
Thank you for your answer.
